Dockerfile

```diff
@@ -1,20 +1,49 @@
-FROM docker.io/alpine:edge AS build
-RUN apk add --no-cache git make musl-dev go nodejs npm gcc g++
+# syntax=docker/dockerfile:1
+
+# Build the web ui from source
+FROM docker.io/node:18 AS build-node
```

Comment:
umm the web ui build uses a node:18 image docker but the application back-end uses alpine goland build.. i dint know if this expected or maybe we must use same image with a node distribution pacakge for?